scipy/stats/_continuous_distns.py

```diff
@@ -5331,15 +5641,23 @@ class argus_gen(rv_continuous):
 
     Notes
     -----
-    The probability density function for `argus` is::
+    The probability density function for `argus` is:
 
-        argus.pdf(x, chi) = chi**3 / (sqrt(2*pi) * Psi(chi)) * x * sqrt(1-x**2) * exp(- 0.5 * chi**2 * (1 - x**2))
+    .. math::
+
+        f(x, \chi) = \frac{\chi^3}{\sqrt{2\pi} \Psi(\chi)} x \sqrt{1-x^2}
+                     \exp(- 0.5 \chi^2 (1 - x^2))
```

Comment:
The old (outdated) error is here:

https://travis-ci.org/scipy/scipy/jobs/273560235#L1718

But the hint is still correct -- you have `\e` but no `r"""`